security-release-process.md

```diff
@@ -0,0 +1,65 @@
+# Gardener Security Release Process
+Gardener is a growing community of volunteers and users. The Gardener community has adopted this security disclosures and response policy to ensure we responsibly handle critical issues.
+
+## Product Security Team (PST)
+Security vulnerabilities should be handled quickly and sometimes privately. The primary goal of this process is to reduce the total time users are vulnerable to publicly known exploits. The Product Security Team (PST) is responsible for organizing the entire response including internal communication and external disclosure but will need help from relevant developers and release managers to successfully run this process. The initial PST will consist of volunteers subscribed to the public reacheble gardener-security@googlegroups.com mailing list. These are the people who have been involved in the initial discussion and volunteered:
+
+* Olaf Beier 
+* Norbert Hamann 
+* Claudia H�lters, (**[@hoeltcl](https://github.com/hoeltcl)**)
+* Oliver Kling 
+* Matthias Sohn, (**[@msohn](https://github.com/msohn)**)
+* Dirk Marwinski, (**[@marwinski](https://github.com/marwinski)**)
+* Vasu Chandrasekhara, (**[@vasu1124](https://github.com/vasu1124)**)
+* Vedran Lerenc, (**[@vlerenc](https://github.com/vlerenc)**)
+* Frederik Thormaehlen, (**[@ThormaehlenFred](https://github.com/ThormaehlenFred)**)
+
+The above list currently includes only people from SAP. But because Gardener will attract as well contributors from outside SAP we decided to not use an SAP internal DL but a public Google group.

```

Comment:
I think these two sentences can be removed

Reply:
done